packages/icons/icons.yml

```
@@ -1586,6 +1595,16 @@
     - asset
   sizes:
     - 32
+- name: asset--movement
```

```suggestion
- name: asset-movement
```

it looks like this extra dash is what's causing the test failures